### DISCUSSION TITLE
how to build web server?

### DISCUSSION THREAD

**Discussion by tggy0509** [community/community#53476] [Programming Help]
how to  build application and DB server

**Comment by samus-aran** [comment]
Thanks for posting in the GitHub Community, @tggy0509 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by davea42** [comment]
Lots of good advice above, but don't be afraid to start small and get confidence as to what works
(here assuming the small start will remain local, not public,  and not viewable by the world).
